interfaces/builtin/unity7.go

```diff
@@ -117,41 +117,189 @@ dbus (receive, send)
+dbus (send)
+    bus=session
+    path=/StatusNotifierItem
+    interface=org.kde.StatusNotifierItem
+    member="New{AttentionIcon,Icon,OverlayIcon,Status,Title,ToolTip}"
+    peer=(name=org.freedesktop.DBus, label=unconfined),
+
+dbus (receive)
+    bus=session
+    path=/StatusNotifierItem
+    interface=org.freedesktop.DBus.Properties
+    member=Get*
     peer=(label=unconfined),
 
+dbus (send)
+    bus=session
+    path=/org/freedesktop/Notifications
+    interface=org.freedesktop.Notifications
+    member="{GetCapabilities,GetServerInformation,Notify}"
+    peer=(name=org.freedesktop.Notifications, label=unconfined),
+
+dbus (receive)
+    bus=session
+    path=/org/freedesktop/Notifications
+    interface=org.freedesktop.Notifications
+    member=NotificationClosed
+    peer=(label=unconfined),
+
+# unity launcher
+dbus (send)
+    bus=session
+    path=/com/canonical/unity/launcherentry/[0-9]*
+    interface=com.canonical.Unity.LauncherEntry
+    member=Update
+    peer=(name=org.freedesktop.DBus, label=unconfined),
+
+dbus (send)
+    bus=session
+    path=/com/canonical/unity/launcherentry/[0-9]*
+    interface=com.canonical.dbusmenu
+    member="{LayoutUpdated,ItemsPropertiesUpdated}"
+    peer=(name=org.freedesktop.DBus, label=unconfined),
+
+dbus (receive)
+    bus=session
+    path=/com/canonical/unity/launcherentry/[0-9]*
+    interface="{com.canonical.dbusmenu,org.freedesktop.DBus.Properties}"
+    member=Get*
+    peer=(label=unconfined),
+
+# This rule is meant to be covered by abstractions/dbus-session-strict but
+# the unity launcher code has a typo that uses /org/freedesktop/dbus as the
+# path instead of /org/freedesktop/DBus, so we need to all it here.
+dbus (send)
+    bus=session
+    path=/org/freedesktop/dbus
+    interface=org.freedesktop.DBus
+    member=NameHasOwner
+    peer=(name=org.freedesktop.DBus, label=unconfined),
+
+# appmenu
+dbus (send)
+    bus=session
+    path=/org/freedesktop/DBus
+    interface=org.freedesktop.DBus
+    member=ListNames
+    peer=(name=org.freedesktop.DBus, label=unconfined),
+
+dbus (send)
+    bus=session
+    path=/com/canonical/AppMenu/Registrar
+    interface=com.canonical.AppMenu.Registrar
+    member="{RegisterWindow,UnregisterWindow}"
+    peer=(label=unconfined),
+
+dbus (send)
+    bus=session
+    path=/com/canonical/AppMenu/Registrar
+    interface=com.canonical.dbusmenu
+    member=UnregisterWindow
+    peer=(label=unconfined),
+
+dbus (receive)
+    bus=session
+    path=/com/canonical/menu/[0-9]*
+    interface="{org.freedesktop.DBus.Properties,com.canonical.dbusmenu}"
+    member="{GetAll,GetLayout}"
+    peer=(label=unconfined),
+
+
 # Lttng tracing is very noisy and should not be allowed by confined apps. Can
 # safely deny. LP: #1260491
 deny /{,var/}run/shm/lttng-ust-* r,
 
-
-# TODO: pull in modern items from ubuntu-unity7-base abstraction, eg, HUD,
-# freedesktop notifications, etc
+# TODO: pull in modern items from ubuntu-unity7-base abstraction, eg, HUD, etc
```

Comment:
This should give you for free HUD items as well, so I guess you can remove it from list.